src/nnue/layers/simd.h

```diff
@@ -99,9 +99,9 @@ namespace Stockfish::Simd {
 
 [[maybe_unused]] static void
 dotprod_m128_add_dpbusd_epi32(int32x4_t& acc, int8x16_t a, int8x16_t b) {
-
     acc = vdotq_s32(acc, a, b);
 }
+
```

Comment:
can you revert the change here please?

Reply:
Fixed. I test and modify code here and there so there might be other places I missed which break the formatting. In that case it's ok for you to correct them during the merge.